src/lib.rs

```diff
@@ -2275,9 +2276,9 @@ pub unsafe trait FromBytes: FromZeros {
 
     /// Interprets the given `bytes` as a `&Self` without copying.
     ///
-    /// If `bytes.len()` does not correspond to a valid length for `Self`, or if
-    /// `bytes` is not aligned to `Self`'s alignment requirement, this returns
-    /// `Err`.
+    /// This method attempts to return a reference to `bytes` interpreted as a
+    /// `Self`. If there are insufficient or excess bytes, or if `bytes` is not
+    /// aligned to `Self`'s alignment requirement, this returns `Err`.
```

Comment:
Here I think the old wording - "valid length for `Self`" (although I'd say size instead of length) - makes more sense. "Insufficient or excess" implies that there's one correct size, but that's not true - there could be many correct sizes.

Reply:
How could there be many correct sizes? Isn't this backed by a `try_cast_into_no_leftover`?

Reply:
Ah, I see your point. There's only one correct size per DST length though.

Reply:
Yeah exactly